demo/build.gradle

```diff
@@ -48,7 +48,7 @@ android {
 dependencies {
 
     // [START_EXCLUDE silent]
-    implementation 'com.google.android.gms:play-services-maps:18.1.0'
+    implementation 'com.google.android.gms:play-services-maps:18.2.0'
```

Comment:
We shouldn't need this line anymore because the library loads the Maps SDK dependency right?